testsuite/MDAnalysisTests/coordinates/test_xdr.py

```diff
@@ -903,13 +908,10 @@ def test_persistent_offsets_readonly(self, tmpdir):
 
         filename = str(tmpdir.join(os.path.basename(self.filename)))
         # try to write a offsets file
-        with (pytest.warns(UserWarning, match="Couldn't save offsets") and
+        with (pytest.warns(UserWarning, match="Couldn't save offsets") or
```

Comment:
Why the change to the tests?

At a minimum, add a comment explaining under which circumstances "or" applies.

Reply:
Ah yes, some of the pytest modifications are not required, I will backtrack the modifications that are not required, this should be one of those, my bad